content/en/guides/hosting/monitoring-usage/org_dashboard.md

```diff
@@ -3,62 +3,95 @@ menu:
+{{% /tab %}}
+{{< /tabpane >}}
+
+## Export user details
+
+{{< tabpane text=true >}}
+{{% tab header="Dedicated or Self-managed" value="dedicated" %}}
+From the **Users** tab, you can export details about how your organization uses W&B in CSV format.
+
+1. Navigate to the **Organization Dashboard** at `https://<org-name>.io/org/dashboard/`. Replace `<org-name>` with your organization name. The **Users** tab opens by default.
+1. Click the action `...` menu next to the **Invite new user user** button.
+1. Click **Export as CSV**. The CSV files that downloads lists details about each user of an organization, such as their user name and email address, the time they were last active, their roles, and more.
+{{% /tab %}}
 
-1. To sort the list of users by **Last Active**, click the column name.
-1. To see details about a user's last activity, hover your mouse over the **Last Active** field for the user.  A tooltip appears that shows when the user was added and how many total days the user has been active.
 
-A user is _active_ if they:
-- log in to W&B.
-- view any page in the W&B App.
-- log runs.
-- use the SDK to track an experiment.
-- interact with the W&B Server in any way.
+{{% tab header="Multi-tenant SaaS Cloud" value="saas" %}}
+1. Navigate to the [Members page](https://wandb.ai/account-settings/wandb/members/).
+1. Click the action `...` menu next to the search field.
+1. Click **Export as CSV**. The CSV files that downloads lists details about each user of an organization, such as their user name and email address, the time they were last active, their roles, and more.
+{{% /tab %}}
+{{< /tabpane >}}
 
 ## View active users over time
-Use the plots in the **Activity** tab to get an aggregate view of how many users have been active over time
+This section shows how to get an aggregate view of how many users have been active over time.
+
+{{< tabpane text=true >}}
+{{% tab header="Dedicated or Self-managed" value="dedicated" %}}
+
+Use the plots in the **Activity** tab to get an aggregate view of how many users have been active over time.
 
 1. Click the **Activity** tab.
-1. The **Total active users** plot shows how many users have been active in a period of time (defaults to 3 months).
+1. The **Total active users** plot shows how many unique users have been active in a period of time (defaults to 3 months).
 1. The **Users active over time** plot shows the fluctuation of active users over a period of time (defaults to 6 months). Hover your mouse over a pointo to see the number of users on that date.
 
 To change the period of time for a plot, use the drop-down. You can select:
 - Last 30 days
 - Last 3 months
 - Last 6 months
 - Last 12 months
-- All time
\ No newline at end of file
+- All time
+
+{{% /tab %}}
+{{% tab header="Multi-tenant SaaS Cloud" value="saas" %}}
+
+Use the plots in the **Organization Dashboard** to get an aggregate view of how many users have been active over time:
```

Comment:
Add a line where the org dashboard is? It's mentioned in a Dedicated tab above, but users on the Multi-tenant SaaS Cloud route might not see it.